docs/Developers/Git_Basics.rst

```diff
@@ -93,6 +104,7 @@ never submit a change that isn't tested), you should check your progress. Type:
     $ git status
 
 It will give you an idea about what files are currently modified.
+It will also tell you which branch you're developing on.
```

Comment:
what about adding this info in the first sentence.
``It will give you an idea about what files are currently modified and also which branch you're developing on.``